tool/site/client-react/src/components/textAreaField.tsx

```diff
@@ -0,0 +1,15 @@
+import React, {CSSProperties, FC} from "react"
+import {Textarea} from "nhsuk-react-components"
+import {Field} from "formik"
+
+export interface TextAreaFieldProps {
+  name: string
+  defaultValue?: string
+  style?: CSSProperties
+}
+
+const TextAreaField: FC<TextAreaFieldProps> = ({name, defaultValue, style}) => (
```

Comment:
nhs has a Text Area component: https://nhsdigital.github.io/nhsuk-react-components/?path=/docs/components-textarea--standard

Reply:
might be more visually appealing

Reply:
It is using the nhs component, this is a wrapper for formik to handle form submission